src/cpp/README.md

```diff
@@ -30,20 +25,75 @@ To add gRPC as a dependency in bazel:
   grpc_deps()
   ```
 
-## cmake
+## CMake
+
+`cmake` is your best option if you cannot use bazel. It supports building on Linux,
+MacOS and Windows (official support) but also has a good chance of working on
```

Comment:
I think visual studio has pretty good builtin support for CMake these days. Can you come up with recommendation what's the best way of including gRPC in a C++ project if you're on Windows and you use Visual Studio to build C++? It would be good to add a note to one of the subsections basically saying "if you're on Windows and using Visual Studio, this is basically option you want to use because it will work well with Visual Studio's cmake support".

Reply:
Adding gRPC as a dependency to your CMake project won't be different on Visual Studio vs. some other generator. We could add some docs about [Visual Studio + CMake integration](https://docs.microsoft.com/en-us/cpp/build/cmake-projects-in-visual-studio?view=vs-2019) in another, more developer-centric section if you think that would be worthwhile.